src/store/reducers/stateCache/stateCache.ts

```diff
@@ -55,6 +55,13 @@ const stateCacheSlice = createSlice({
         (route) => route.path !== RoutePath.SET_PASSCODE
       );
     },
+    logout: (state) => {
+      state.routes = [{ path: RoutePath.PASSCODE_LOGIN }, ...state.routes];
+      state.authentication = {
+        ...state.authentication,
+        loggedIn: false,
+      };
+    },
```

Comment:
Hmm, I wonder if we should stop doing things like polling KERIA when we're logged out. At least right now we don't have push notifications when something happens anyway so it's not that useful.

Right now it actually starts polling when the app is launched but this could be adjusted too. It's good to have the agent started and everything loaded while we are in the login page though, so that the app is "ready" for when they type in their pin code and seamless.

But if its in the background it seems overkill. Since KERIA is in the cloud, all we really need to do stop polling KERIA (`signifyNotificationService.ts`) and then re-enable when the app is active.

Reply:
There is also `pauseQueueIncomingRequest` which is `false` until you login - so the logout needs to handle pausing this too.

Reply:
we can detect if the app is the background by using @capacitor/app.

https://capacitorjs.com/docs/next/apis/app#getstate